hack/lib/mutil-arch.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Use multiarch/qemu-user-static that is to enable an execution of different multi-architecture containers
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+# Add buildx plugin if the system doesn't have it.
+docker_buildx=~/.docker/cli-plugins/docker-buildx
+if [ ! -x $docker_buildx ];then
+  mkdir -pv $(dirname $docker_buildx)
+  arch=$(dpkg --print-architecture)
```

Comment:
dpkg only available in debain like system, use `uname -m`

Reply:
delete `install buildx`